### DISCUSSION TITLE
Why is PyYAML package version between >=3.10,<5.5 ?

### DISCUSSION THREAD

**Discussion by sadikkuzu** [aws/aws-cli#7534] [Q&A] [answered]
https://github.com/aws/aws-cli/blob/aa82eabdd677bbd02375452aaa577b8fd89f2c87/setup.cfg#L9

On the other hand, [PyYAML has 6.0 version](https://pypi.org/project/PyYAML/#history).

**Comment by RyanFitzSimmonsAK** [comment, marked as answer]
Hi @sadikkuzu, thanks for reaching out.  There is an issue tracking updates to pinned versions for dependencies, including PyYAML, found here: [5943](https://github.com/aws/aws-cli/issues/5943).  While we can't guarantee any specific timeline, PyYAML 6.x has been mentioned in the tracking thread, as well as in other issues.  I'd encourage you to 👍 the original post and posts referring to PyYAML 6.x, and to bring up your use case as a comment.
